### PR TITLE
Fix the padding top as that caused the hide issue login screen

### PR DESCRIPTION
**Pull request by mahmed0715**
@sagararyal please check

### CODE REVIEW

**Comment by sagararyal** [comment]
I have tested with iPhone 5s, and iPhone 11
:works fine.

Can someone test this with small android devices? @mahmed0715 @harshvitra

**Comment by norbertschuler** [comment]
@sagararyal what to test? What was wrong?

File: app/components/Authentication/Login/index.native.js [hunk 1]
```diff
@@ -157,6 +157,7 @@ export default class Login extends Component {
                       labelTextStyle={{ fontFamily: 'OpenSans-Regular' }}
                       titleTextStyle={{ fontFamily: 'OpenSans-SemiBold' }}
                       affixTextStyle={{ fontFamily: 'OpenSans-Regular' }}
+                      autoCorrect="false"
```
Comment (norbertschuler) [inline app/components/Authentication/Login/index.native.js]: Crashes on Android 6 on my Huawei Honor 6:

Screenshot_2020-02-05-21-22-02

Comment (norbertschuler) [reply app/components/Authentication/Login/index.native.js]: Fixed it.

**Pull request merged**